relay-dynamic-config/src/global.rs

```diff
@@ -79,6 +74,16 @@ pub struct Options {
     )]
     pub cardinality_limiter_mode: CardinalityLimiterMode,
 
+    /// Sample rate for Cardinality Limiter Sentry errors.
+    ///
+    /// Rate needs to be between `0.0` and `1.0`.
+    /// If set to `1.0` all cardinality limiter rejections will be logged as a Sentry error.
+    #[serde(
+        rename = "relay.cardinality-limiter.error-sample-rate",
+        skip_serializing_if = "is_default"
+    )]
+    pub cardinality_limiter_error_sample_rate: f32,
```

Comment:
We could theoretically make this `#[cfg(processing)]` (probably other cardinality limiter options as well).

Reply:
Not a fan of this, it would just end up being collected in the fallback hashmap instead